common/src/chain/tokens/rpc.rs

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2022 RBB S.r.l
+// opensource@mintlayer.org
+// SPDX-License-Identifier: MIT
+// Licensed under the MIT License;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://github.com/mintlayer/mintlayer-core/blob/master/LICENSE
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use serialization::Decode;
+use serialization::Encode;
+
+// use crate::primitives::VersionTag;
+use crate::{
+    chain::{Block, Transaction},
+    primitives::{Amount, Id},
+};
+
+use super::TokenId;
+
+#[derive(Debug, Clone, Encode, Decode, serde::Serialize, serde::Deserialize)]
+pub struct RPCTokenInfoV1 {
```

Comment:
Versioning is not arbitrary. We only version things that are part of consensus. The reason for versioning is to be able to support protocol upgrades. If I'm not mistaken, this struct is not used in anything protocol-related. This is just used as an abstraction layer for token info. Meaning, if we change it, it doesn't break the blockchain by failing to deserialize old things, it just changes the printed information in RPC... well, who cares? If that's the case, please remove the V1 from the name.

Reply:
Ok, no problem. Thanks for explanation

Reply:
Removed versioning